common/datavalues/src/columns/struct_/mutable.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_arrow::arrow::bitmap::MutableBitmap;
+use common_exception::ErrorCode;
+use common_exception::Result;
+
+use crate::prelude::*;
+
+pub struct MutableStructColumn {
+    pub data_type: DataTypeImpl,
```

Comment:
Struct should have multiple inner data types.

Reply:
Yes，but we only use the StructType itself here.